test/MeasureToolbox/integrals.jl

```diff
@@ -26,13 +26,23 @@
     @testset "generate_integral_data (Gauss-Lobatoo)" begin
         @test generate_integral_data(t, 0, 1, GaussLobatto()) isa DiscreteMeasureData
     end
+
+    # test generate_integral_data (FEGaussLobatto)
+    @testset "generate_integral_data (FEGaussLobatto)" begin
+        @test generate_integral_data(t, 0, 1, FEGaussLobatto()) isa FunctionalDiscreteMeasureData
+    end
+
```

Comment:
On this test and the others, we should check the contents of the measure data outputs to make sure they are outputting what we expect.